config/src/config/network_config.rs

```diff
@@ -96,6 +98,10 @@ pub struct NetworkConfig {
     pub max_outbound_connections: usize,
     // Maximum number of outbound connections, limited by PeerManager
     pub max_inbound_connections: usize,
+    // Maximum number of bytes/s for an IP
+    pub inbound_ip_byte_bucket_rate: usize,
+    // Maximum burst of bytes for an IP
+    pub inbound_ip_byte_bucket_size: usize,
```

Comment:
maybe make this an option -- if both aren't set than we don't instantiate the rate limiter?

alternatively make this an optional struct, since we'll need the same values for writing

Reply:
Optional struct done.

It's just kinda annoying because now there's a struct in common (for the rate limiter), and one in config (for the config)